### ISSUE TITLE
Strange output

### ISSUE THREAD

**Issue by zdila** [75lb/renamer#66]
I run renamer like this:

```sh
renamer --find '/\\bcjs\\b/' --replace js dist/*cjs*
```

The output is:

```
, , , {, ,} green ✔︎ dist/leaflet-controller.umd.cjs → dist/leaflet-controller.umd.js 
, , , {, ,} green ✔︎ dist/leaflet-controller.umd.cjs.map → dist/leaflet-controller.umd.js.map 
, , , {, ,} green ✔︎ dist/leaflet.umd.cjs → dist/leaflet.umd.js 
, , , {, ,} green ✔︎ dist/leaflet.umd.cjs.map → dist/leaflet.umd.js.map 
, , , {, ,} green ✔︎ dist/maplibregl-controller.umd.cjs → dist/maplibregl-controller.umd.js 
, , , {, ,} green ✔︎ dist/maplibregl-controller.umd.cjs.map → dist/maplibregl-controller.umd.js.map 
, , , {, ,} green ✔︎ dist/maplibregl.umd.cjs → dist/maplibregl.umd.js 
, , , {, ,} green ✔︎ dist/maplibregl.umd.cjs.map → dist/maplibregl.umd.js.map 
, , , {, ,} green ✔︎ dist/maptilersdk.umd.cjs → dist/maptilersdk.umd.js 
, , , {, ,} green ✔︎ dist/maptilersdk.umd.cjs.map → dist/maptilersdk.umd.js.map 
, , , {, ,} green ✔︎ dist/openlayers-controller.umd.cjs → dist/openlayers-controller.umd.js 
, , , {, ,} green ✔︎ dist/openlayers-controller.umd.cjs.map → dist/openlayers-controller.umd.js.map 
, , , {, ,} green ✔︎ dist/openlayers.umd.cjs → dist/openlayers.umd.js 
, , , {, ,} green ✔︎ dist/openlayers.umd.cjs.map → dist/openlayers.umd.js.map 
, , , {, ,} green ✔︎ dist/react.umd.cjs → dist/react.umd.js 
, , , {, ,} green ✔︎ dist/react.umd.cjs.map → dist/react.umd.js.map 
, , , {, ,} green ✔︎ dist/vanilla.umd.cjs → dist/vanilla.umd.js 
, , , {, ,} green ✔︎ dist/vanilla.umd.cjs.map → dist/vanilla.umd.js.map 
```

It looks like some ANSI escape sequences don't work. We also use `concurrently` which also does colored output and that works.

**Comment by 75lb** [comment]
Reproduced in Terminal and iTerm2 - thanks for letting me know.. looks like there has been a breaking change somewhere in the dep tree, will look into it..

**Comment by 75lb** [comment]
Fixed and released in renamer v5.0.1 - let me know if you find anything else, thanks.
